app/components/admin/AddProductForm.tsx

```diff
@@ -178,7 +178,7 @@ export default function AddProductForm() {
               <Button
                 onClick={handleAddAttribute}
                 text='Add Attribute +'
-                className='rounded-t-3xl w-full mt-5 bg-slate-400 hover:bg-slate-500'
+                className='rounded-t-3xl  w-full mt-5 bg-slate-400 hover:bg-slate-500'
```

Comment:
hmm use a better way to add styles with tailwind, creating a config.js file or something: https://tailwindcss.com/docs/adding-custom-styles